### ISSUE TITLE
Use spare pins to wire up extra keys?

### ISSUE THREAD

**Issue by kindfulkirby** [No0ne/ps2x2pico#56]
The Pico seems to have a lot of unused GPIOs, how difficult would it be to use them for extra keyboard keys or even mouse actions?

For example: You have a small keyboard and wanted to add a numpad and don't mind making your own, so you wire up some key switches to the Pico's spare GPIOs.

It might be useful if it could behave differently based on modifier keys from the main keyboard, for example, pressing CTRL on the main keyboard to have the numpad "arrows" move the mouse cursor.

If this out of scope, how hard do you think it would be to implement in a fork?
Or would it have to be a separate USB device, with a hub?

**Comment by No0ne** [comment]
Yes that should all be possible. In fact if you connect a keyboard without numpad and then an extra usb numpad together with a hub this should already work.
